.travis.yml

```diff
@@ -245,7 +245,7 @@ install:
   - perl Makefile.PL
   - make regen
 # ensure allfiles are git up-to-date [fail if make regen alters some files]
-  - git diff --quiet
+  - git diff --quiet || ( git diff | cat; [ "$TRAVIS_OS_NAME" = "windows" ] )
```

Comment:
this reads weird to me,
should not this be something like 
`- [ "$TRAVIS_OS_NAME" = "windows" ] && git diff --quiet ||:`
`- [ "$TRAVIS_OS_NAME" != "windows" ] && git diff`